src/home/homework-luks.c

```diff
@@ -2616,7 +2616,7 @@ static int ext4_offline_resize_fs(
                 return r;
         if (r == 0) {
                 /* Child */
-                execlp("e2fsck" ,"e2fsck", "-fp", setup->dm_node, NULL);
+                execlp("e2fsck", "e2fsck", "-fp", setup->dm_node, NULL);
                 log_open();
```

Comment:
Hmm, while at it, I think this should be switched to `FORK_REOPEN_LOG`

Reply:
Maybe? But I'd like to leave such change to another PR, and I'd like to keep this PR super trivial.